ZooRoyal/coding-standard
# Make coding-standard robust against execution from other paths

* Ask the surrounding git repo for it's root
* Distinguish between root directory of code to check and directory
  where coding-standard is installed

# How to check that it works?

* Clone Coding-Standard to filesystem (e.g. /a/cs)
* Composer install
* Switch to another non empty directory outside of coding-standard (e.g. /b)
* Call coding-standard from here (e.g. ../a/src/bin/coding-standard sca:all)
* All PHP-Checks should work as expected
